boards/328p16m.json

```diff
@@ -22,4 +25,4 @@
   },
   "url": "http://wiki.microduinoinc.com/Microduino-Module_Core",
   "vendor": "Microduino"
-}
+}
```

Comment:
Please add a new line at EOF for each manifest

Reply:
Done